worldengine/model/world.py

```diff
@@ -852,7 +851,67 @@ def is_chaparral(self, pos):
     #
 
     def n_actual_plates(self):
-        return self.layers['plates'].data.max() + 1
+        return self.plates.data.max() + 1
+
+    #
+    # Accessors
+    #
+
+    @property
+    def elevation(self):
+        return self.layers['elevation']
```

Comment:
Yes, sure but if the property reader return a layer the property setter should accept a layer (otherwise it would be really weird from the user point of view) and so the user would have to instantiate those Layer classes.